CONTRIBUTING.md

```diff
@@ -6,9 +6,13 @@ To contribute to the guide, fork this repository and make the necessary ammendme
 
 ## Self-hosting
 
-To host the guide on GitHub Pages, the quickest way is by enabling it in the settings of GitHub. Set GitHub Pages to render from the `master` branch (refer [here](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)). Once this is done, GitHub Pages will take a while to upload your changes and it will soon be available at `https://<username>.github.io/git-guide`.
+~~To host the guide on GitHub Pages, the quickest way is by enabling it in the settings of GitHub. Set GitHub Pages to render from the `master` branch (refer [here](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)). Once this is done, GitHub Pages will take a while to upload your changes and it will soon be available at `https://<username>.github.io/git-guide`.~~
 
-**Note*** it is possible to use a separate documentation generator for this but for simplicity sake, we will be using the default GitHub Pages template.
+~~**Note*** it is possible to use a separate documentation generator for this but for simplicity sake, we will be using the default GitHub Pages template.~~
+
+Update as of **01 August 2020** we are now using Gatsby.js to host our guide on github pages.  
+
+You can find out more information on how to self-host the website [here.](./site/README.md)
```

Comment:
Not sure to leave in the old self-host instructions

Reply:
They would actually have to enable this option in the settings regardless of the self-host option. That's a prerequisite to actually viewing the site on GitHub pages.

Reply:
Ah I see, I thought the development set up was how you could see the website locally.  
I guess you would still need the `gh-pages` branch to self-host it.